posthog/cdp/templates/snapchat_ads/template_snapchat_ads.py

```diff
@@ -0,0 +1,141 @@
+from posthog.cdp.templates.hog_function_template import HogFunctionTemplate
+
+template: HogFunctionTemplate = HogFunctionTemplate(
+    status="alpha",
+    type="destination",
+    id="template-snapchat-ads",
+    name="Snapchat Ads Conversions",
+    description="Send conversion events to Snapchat Ads",
+    icon_url="/static/services/snapchat-ads.png",
```

Comment:
For the future - we can just make these "snapchat" if the logo isn't specific to ads.